tests/playwright/pages/editor-page.ts

```diff
@@ -322,12 +322,20 @@ export default class EditorPage extends BasePage {
 		await this.page.locator( '.elementor-control-' + controlId + ' .elementor-slider-input input' ).fill( value.toString() );
 	}
 
+	async setTextControlValue( controlId: string, value: string ) {
+		await this.page.locator( `.elementor-control-${ controlId } input` ).fill( value.toString() );
+	}
+
+	async setTextareaControlValue( controlId: string, value: string ) {
+		await this.page.locator( `.elementor-control-${ controlId } textarea` ).fill( value.toString() );
+	}
+
 	async setNumberControlValue( controlId: string, value: string ) {
 		await this.page.locator( `.elementor-control-${ controlId } input >> nth=0` ).fill( value.toString() );
 	}
 
 	async setChooseControlValue( controlId: string, value: string ) {
-		await this.page.locator( `.elementor-control-${ controlId } .${ value }` ).click();
+		await this.page.locator( `.elementor-control-${ controlId } input[value="${ value }"]` ).click();
```

Comment:
looks like selector doesn't work here. Some tests are failing

Reply:
Yes, I'm on it.

Reply:
Reverted. I'll fix this in a separate PR.